app/web/readme.md

```diff
@@ -24,6 +24,7 @@ If you don't want `nvm` to be run when you open your shell, you can add `--no-us
 On macOS you can alternatively install dependencies via homebrew. You can get nvm with `brew install nvm`, or `nodejs` directly with `brew install node@20`.
 
 You also need `git`, `wget`, and `tar`, they should be preinstalled or be available from the usual places.
```

Comment:
Please also update this line and any other instances!

Reply:
I think I'll leave the ones that are actually in scripts since they are already working, even though things shouldn't break in theory...